src/Control/Lens/At.hs

```diff
@@ -286,9 +286,15 @@ instance Ixed (Tree a) where
 
 type instance IxValue (Seq a) = a
 instance Ixed (Seq a) where
+#if MIN_VERSION_containers(0,5,8)
+  ix i f m = case Seq.lookup i m of
+    Just x -> f x <&> \x' -> Seq.update i x' m
+    Nothing -> pure m
+#else
   ix i f m
     | 0 <= i && i < Seq.length m = f (Seq.index m i) <&> \a -> Seq.update i a m
```

Comment:
Is here impossible to `seq` things to avoid thunk, Would

```
let x = Seq.index m i `seq` (f x <&> ...)
```

do anything?

Reply:
Using `seq` like that would force *too much*, forcing not only the lookup but also the element itself.

Reply:
I see. as later then `0.5.8` `containers` are bundled with GHC-8.2, I don't think we need to do here anything. (If someone have problems due this lazyness, and they could upgrade GHC or/and containers)

Reply:
@phadej Do you mean we shouldn't worry about the leak for older `containers`? I agree. Trying to plug it in that context will be very inefficient when `f` is actually strict.